Explorer/Assets/DCL/LOD/Pools/Utils/LODGroupPoolUtils.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using DCL.Optimization.Pools;
+using UnityEngine;
+
+namespace DCL.LOD.Systems
+{
+    public static class LODGroupPoolUtils
+    {
+        public static void ReleaseLODGroup(LODGroup lodGroup)
+        {
+            GenerateDefaultLODs(lodGroup);
+        }
+
+        public static void GenerateDefaultLODs(LODGroup lodGroup)
+        {
+            lodGroup.name = "LODGroup";
+            var lod0 = new UnityEngine.LOD();
+            var lod1 = new UnityEngine.LOD();
+            lod0.renderers = Array.Empty<Renderer>();
```

Comment:
It could be a moment to release these arrays to the pool if you decided to pool arrays of renderers